x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/analytics/process/AnalyticsControlMessageWriter.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+package org.elasticsearch.xpack.ml.analytics.process;
+
+import org.elasticsearch.xpack.ml.process.writer.AbstractControlMsgWriter;
+import org.elasticsearch.xpack.ml.process.writer.LengthEncodedWriter;
+
+import java.io.IOException;
+
+public class AnalyticsControlMessageWriter extends AbstractControlMsgWriter {
+
+    /**
+     * This must match the code defined in the api::CDataFrameAnalyzer C++ class.
```

Comment:
Since the constant has a different name in the C++ it would be worth stating the C++ name here.